README.md

```diff
@@ -40,9 +40,12 @@ The downsides of using YouTube IFrame Player API are:
 /**
  * A factory function used to produce an instance of YT.Player and queue function calls and proxy events of the resulting object.
  *
- * @param {HTMLElement|String} elementId Either An existing YT.Player instance,
+ * @param {Object|HTMLElement|String} elementId Either An existing YT.Player instance,
  * the DOM element or the id of the HTML element where the API will insert an <iframe>.
  * @param {YouTubePlayer~options} options See `options` (Ignored when using an existing YT.Player instance).
+ * @param {boolean} strictState Experimental: A flag designating whether or not to wait for
```

Comment:
Please remove the `Experimental: `. These "experimental" warnings mean nothing when the package is released without a flair. We can always make a major release if this needs to be removed.